lib/cryptoexchange/exchanges/novaexchange/models/market_pair.rb

```diff
@@ -0,0 +1,14 @@
+module Cryptoexchange::Exchanges
+  module Novaexchange
+    module Models
+      class MarketPair < Cryptoexchange::Models::MarketPair
+        attr_accessor :name
+
+        def initialize(params={})
+          super
+          @name = params[:name]
```

Comment:
Well, the easy way out is to hardcode this special case in the code, but that's very hacky.